src/routes/Wallets/routes/Backup/WalletsBackupView.js

```diff
@@ -77,10 +105,9 @@ class WalletsBackupView extends Component<Props> {
           {(currentStep === STEPS.PRIVATE) && (
             <div className='wallet-private-step'>
               <form className='form' onSubmit={downloadToTxt}>
-                <MnemonicPhrase
+                <CopyableField
                   copy={copyToClipboard}
-                  download={downloadToTxt}
-                  mnemonic={data}
+                  value={isMnemonic ? data : `${data.substr(0, 33)} ${data.substr(33)}`}
```

Comment:
I think it can be moved to some separated variable with description what is it ``${data.substr(0, 33)} ${data.substr(33)}``?

Reply:
definitely, 33 needs a naming.